source/extensions/transport_sockets/tls/ssl_socket.cc

```diff
@@ -22,7 +22,7 @@ namespace Tls {
 
 namespace {
 
-constexpr absl::string_view NotReadyReason{"TLS error: Secret is not delivered via SDS"};
+constexpr absl::string_view NotReadyReason{"TLS error: Secret is not supplied by SDS"};
```

Comment:
Would be nice if there was a test that covers this?

Reply:
done